packages/logs/src/logsEventsFormat.ts

```diff
@@ -0,0 +1,110 @@
+export type LogsEventsFormat = LogSchema | LogWithRumSchema
```

Comment:
> Why having two different types, with and without rum?

It is to reflect that depending on if logs is installed alongside rum, we can have these two different types.

> I would not use the Format or Schema suffixes. LogEvent or Log seems more appropriate.
> LogsEvents should not be plural since it represent a single event.

The idea was to be consistent with what we've done for rum, which was generated from naming used by rum events format files.
On the RUM side, it is handy to have a different naming than the one we are using internally.

what would you do?

Reply:
> > Why having two different types, with and without rum?
> 
> It is to reflect that depending on if logs is installed alongside rum, we can have these two different types.
I think that it doesn't brings much value at the cost of exposing 4 types instead of 1. I would expose only one type and make RUM internal context fields optional.

> 
> > I would not use the Format or Schema suffixes. LogEvent or Log seems more appropriate.
> > LogsEvents should not be plural since it represent a single event.
> 
> The idea was to be consistent with what we've done for rum, which was generated from naming used by rum events format files.
> On the RUM side, it is handy to have a different naming than the one we are using internally.
> 
> what would you do?

At the very least, I'd keep only Format or Schema, using both seems confusing.

For the RUM conflicting types, I'd be in favor of renaming the rum internal types, and exposing simpler type names (`RumEvent, RumActionEvent...`) publicly.

Reply:
> I think that it doesn't brings much value at the cost of exposing 4 types instead of 1. I would expose only one type and make RUM internal context fields optional.

ok, no expressed need for now, let's simplify.

Reply:
about naming, what would you think of:
- LogsEvent
- RumEvent = RumActionEvent | RumErrorEvent | ...
- RawRumEvent = RawRumActionEvent | RawRumErrorEvent | ...

Reply:
Looks great!